vpr/src/route/rr_graph_uxsdcxx_serializer.h

```diff
@@ -1534,6 +1537,10 @@ class RrGraphSerializer final : public uxsd::RrGraphBase<RrGraphContextTypes> {
         VTR_ASSERT(read_rr_graph_name_ != nullptr);
         read_rr_graph_filename_->assign(read_rr_graph_name_);
 
+        /* Update RRGraph viewer with new data pointers */
+        rr_graph_->set_internal_data(rr_nodes_,
```

Comment:
See comment above, but this sounds like we want to avoid that and consider making a fresh `rr_graph_` with pointers that are only assigned once.

Reply:
Agree. Removed.